guides/fields/instrumentation.md

```diff
@@ -1,50 +0,0 @@
----
-layout: guide
-doc_stub: false
-search: true
-section: Fields
-title: Instrumentation
-desc: Wrap and modify resolution behavior at schema build-time
-index: 3
```

Comment:
@rmosolgo : Do we need to update index of other documentation after removing this?

Reply:
No, we don't need to update it. It just `.sort {...}`s by this, so it's ok to have a gap!